### PR TITLE
Add tab completion support in launcher.

### PR DESCRIPTION
**Pull request by Chuvi-w**

### CODE REVIEW

File: rehlds/public/rehlds/rehlds_api.h [hunk 1]
```diff
@@ -283,6 +283,8 @@ struct RehldsFuncs_t {
 	void(*SV_UpdateUserInfo)(IGameClient *pGameClient);
 	bool(*StripUnprintableAndSpace)(char *pch);
 	void(*Cmd_RemoveCmd)(const char *cmd_name);
+	struct cmd_function_s*(*Cmd_GetFirstCmd)(void);
```
Comment (LevShisterov) [inline rehlds/public/rehlds/rehlds_api.h]: Please use `cmd_function_t` here.

**Comment by LevShisterov** [comment]
Would you like to complete TODO in ReceiveTab and add to this PR?

File: rehlds/public/rehlds/rehlds_api.h (hunk 1 above)
Comment (Chuvi-w) [inline rehlds/public/rehlds/rehlds_api.h]: Done?

**Comment by Chuvi-w** [comment]
Todo about competion common chars? Not shure, that I'll be able to compete this today.

**Comment by LevShisterov** [comment]
This PR is put on hold. There is a better idea to add IBaseSystem to engine.

**Pull request merged**